hack/testing-olm-upgrade/upgrade-common

```diff
@@ -397,15 +397,8 @@ discover_versions(){
   major_version="$(echo $full_version | cut -d'.' -f1)"
   minor_version="$(echo $full_version | cut -d'.' -f2)"
 
-  # we shouldn't run into this yet... maybe for 6.0 but we should like
-  # add the 'Replaces' spec field and use that for the previous_version
-  if [[ $minor_version -eq 0 ]]; then
-    log::info "Will be unable to calculate the previous_version since our minor version is 0"
-    exit 0
-  fi
-
   export version="$(echo $major_version.$minor_version)"
-  get_latest_previous_version $version
+  get_latest_previous_version "$(echo $major_version.$minor_version)"
```

Comment:
aren't these the same?

Reply:
These are the same, this is a hindsight from cherry-picking the fix from `release-5.0`.